frontend/apps/reader/modules/readerfooter.lua

```diff
@@ -807,6 +868,61 @@ function ReaderFooter:addToMainMenu(menu_items)
                 end,
                 keep_menu_open = true,
             },
+            {
+                text = _("Maximum width of items"),
+                sub_item_table = {
+                    {
+                        text_func = function()
+                            return T(_("Book title (%1%)"), self.settings.book_title_max_width_pct)
+                        end,
+                        callback = function(touchmenu_instance)
+                            local SpinWidget = require("ui/widget/spinwidget")
+                            local items = SpinWidget:new{
+                                width = Screen:getWidth() * 0.6,
+                                value = self.settings.book_title_max_width_pct,
+                                value_min = 10,
+                                value_step = 5,
+                                value_hold_step = 20,
+                                value_max = 100,
+                                title_text =  _("Maximum width"),
+                                text = _("Maximum book title width in percentage of screen width"),
+                                callback = function(spin)
+                                    self.settings.book_title_max_width_pct = spin.value
+                                    self:refreshFooter(true, true)
+                                    if touchmenu_instance then touchmenu_instance:updateItems() end
+                                end
+                            }
+                            UIManager:show(items)
+                        end,
+                        keep_menu_open = true,
+                    },
+                    {
+                        text_func = function()
+                            return T(_("Current chapter (%1%)"), self.settings.book_chapter_max_width_pct)
+                        end,
+                        callback = function(touchmenu_instance)
+                            local SpinWidget = require("ui/widget/spinwidget")
+                            local items = SpinWidget:new{
+                                width = Screen:getWidth() * 0.6,
+                                value = self.settings.book_chapter_max_width_pct,
+                                value_min = 10,
+                                value_step = 5,
+                                value_hold_step = 20,
+                                value_max = 100,
+                                title_text =  _("Maximum width"),
+                                text = _("Maximum book current chapter width in percentage of screen width"),
```

Comment:
Another book current chapter, but I'm not sure the current is really necessary here.
```suggestion
                                text = _("Maximum current chapter width in percentage of screen width"),
```
```suggestion
                                text = _("Maximum chapter width in percentage of screen width"),
```